packages/react-components/src/components/ChatMessage/MyMessageComponents/ChatMessageComponentAsRichTextEditBox.tsx

```diff
@@ -207,7 +207,6 @@ export const ChatMessageComponentAsRichTextEditBox = (
       insertImagesToContentString(textValue, imageUploadsInProgress, (content) => {
         onSubmit(content, /* @conditional-compile-remove(file-sharing-acs) */ attachmentMetadata || []);
       });
-      return;
```

Comment:
Won't we call onSubmit twice in this case(on line 208 and 216)?

Reply:
Good catch! Thanks.